tests/test_all_fields.py

```diff
@@ -18,27 +18,20 @@
 #             Fields that are consistently missing during replication
 #             Original Ticket [https://stitchdata.atlassian.net/browse/SRCE-4736]
 KNOWN_MISSING_FIELDS = {
-    'customers':{
-        'tax_ids',
-    },
+    'customers':set(),
     'subscriptions':{
         'payment_settings',
         'default_tax_rates',
         'pending_update',
         'automatic_tax',
     },
     'products':{
-        'skus',
         'tax_code',
     },
     'invoice_items':{
         'price',
     },
-    'payouts':{
-        'application_fee',
-        'reversals',
-        'reversed',
-    },
+    'payouts':set(),
```

Comment:
As per the [API doc](https://stripe.com/docs/api/payouts/object), the newest API version that is 2020-07-27, doesn't contain these missing fields for payouts object, hence removed the fields from the dictionary